python/lsst/meas/astrom/astrom.py

```diff
@@ -254,11 +259,11 @@ def useKnownWcs(self, sources, wcs=None, exposure=None, filterName=None, imageSi
         probably also want to turn OFF 'calculateSip'.
 
         This involves searching for reference sources within the WCS
-        area, and matching them to the given 'sources'.  If
+        area, and matching them to the given 'sourceCat'.  If
```

Comment:
On not separating the commits: fine.

On plural variable names vs. appending types: I generally disagree (I think including the types makes the code less readable and more fragile, as most containers are substitutable and we do sometimes substitute them).  But our coding standards are indeed on your side, and the most I'd advocate changing them in this respect would be to just let individual developers decide.